src/components/side_panel/chart/building_blocks/generic_side_panel/config_panel.ts

```diff
@@ -83,7 +97,9 @@ export class GenericChartConfigPanel extends Component<Props, SpreadsheetChildEn
   }
 
   get dataSetsHaveTitleLabel(): string {
-    return _t("Use row %s as headers", this.calculateHeaderPosition() || "");
+    return this.datasetOrientation === "rows"
+      ? _t("Use col %s as headers", numberToLetters(this.calculateHeaderPosition() || 0))
```

Comment:
```suggestion
      ? _t("Use col %(column_name)s as headers", { column_name: numberToLetters(this.calculateHeaderPosition() || 0) })
```
It's easier for translators (https://docs.google.com/presentation/d/19dmolYhNiAGAkUBxnbpcfzfX-QXLsU87Em7s_jlnXxA/edit#slide=id.g2ccdc0e95c1_0_54)